lib/FFI/CheckLib.pm

```diff
@@ -74,6 +106,10 @@ else
     \@DynaLoader::dl_library_path;
   };
   die $@ if $@;
+  @extra_paths = _darwin_extra_paths() if $os eq 'darwin';
+}
+foreach my $path (@extra_paths) {
+  push @$system_path, $path unless any { $_ eq $path } @$system_path;
```

Comment:
ah, I just noticed we can't modify $system_path here, because that will modify PATH on Windows and @DynaLoader::dl_library_path on elsewhere.  Instead please add it to the my `@path` here:

https://github.com/cdalvaro/FFI-CheckLib/blob/ec9b1527de929da5d678c6559a0cfb7590e60203/lib/FFI/CheckLib.pm#L338-L342

I don't think it should override `$args{systempath}` if that is passed in.

Reply:
Done!